.travis.yml

```diff
@@ -28,6 +28,6 @@ install:
   - python setup.py install
   - cd ../..
 script:
-  - python setup.py nosetests -s
+  - python setup.py test
```

Comment:
This doesn't work. See the latest Travis log which says "Ran 0 tests".

Reply:
This is working locally but not on travis. Will debug tomorrow and leave it in progress for now